client/my-sites/add-ons/components/add-ons-grid.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled';
+import AddOnCard from './add-ons-card';
+import type { AddOnMeta } from '../hooks/use-add-ons';
+
+interface Props {
+	addOns: ( AddOnMeta | null )[];
+	actionPrimary?: {
+		text: string;
```

Comment:
We may want to pass in `translate` outputs as for `text`. For this, we'll need to allow the `ReactChild` type.